recipe/meta.yaml

```diff
@@ -10,33 +10,43 @@ source:
   sha256: eebca7f587b1761ce43a98ffd5327f2b600b9aa5bb0a77b947687f503ad43bc0
 
 build:
-  noarch: python
+  skip: true  # [py<38]
   script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
-  number: 1
+  number: 0
 
 requirements:
   host:
-    - python >=3.8
+    - python
     - setuptools
     - pip
+    - wheel
   run:
-    - python >=3.8
+    - python
     - cryptography
+    - pycryptodome
```

Comment:
```suggestion
```
pycryptodome is optional: https://github.com/authlib/joserfc/blob/0.9.0/pyproject.toml#L31